scripts/clean_hcfiles.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+#
+#  clean_hcfiles.sh
+#
+#  Script to clean Medley directory after running do_hcfiles.sh.
+#  Removes pdf files and index.html files created by do_hcfiles.sh.
+#
+#  Caution: uses git clean - so it will delete any untracked files in
+#  the Medley directory tree.
+#
+#  FGH 2024-07-15
+#
+#  Copyright 2024 Interlisp.org
+#
+
+main() {
+    MEDLEYDIR=$(cd "${SCRIPTDIR}/.." && pwd)
+    export MEDLEYDIR
+    cd "${MEDLEYDIR}"
+    git clean -f
+    find . -iname index.html.~\*~ -exec rm {} \;
+    find . -iname \*.pdf.~\*~ -exec rm {} \;
```

Comment:
There are a couple of .PDF files that are checked into master or loops. MAKEHCFILES adds ".pdf" lower case. Simplest is to change "-iname" to "name".